soccer/src/soccer/ui/main_window.cpp

```diff
@@ -1146,3 +1170,154 @@ void MainWindow::updateDebugLayers(const LogFrame& frame) {
         _ui.debugLayers->sortItems();
     }
 }
+
+/**
+ * The following methods are event listeners for the manual position assignments.
+ * TODO: implement all of these
+ */
+void MainWindow::on_positionReset_0_clicked() { onResetButtonClicked(0); }
+
+void MainWindow::on_positionReset_1_clicked() { onResetButtonClicked(1); }
+
+void MainWindow::on_positionReset_2_clicked() { onResetButtonClicked(2); }
+
+void MainWindow::on_positionReset_3_clicked() { onResetButtonClicked(3); }
+
+void MainWindow::on_positionReset_4_clicked() { onResetButtonClicked(4); }
+
+void MainWindow::on_positionReset_5_clicked() { onResetButtonClicked(5); }
+
+void MainWindow::on_positionReset_6_clicked() { onResetButtonClicked(6); }
+
+void MainWindow::on_positionReset_7_clicked() { onResetButtonClicked(7); }
+
+void MainWindow::on_positionReset_8_clicked() { onResetButtonClicked(8); }
+
+void MainWindow::on_positionReset_9_clicked() { onResetButtonClicked(9); }
+
+void MainWindow::on_positionReset_10_clicked() { onResetButtonClicked(10); }
+
+void MainWindow::on_positionReset_11_clicked() { onResetButtonClicked(11); }
+
+void MainWindow::on_positionReset_12_clicked() { onResetButtonClicked(12); }
+
+void MainWindow::on_positionReset_13_clicked() { onResetButtonClicked(13); }
+
+void MainWindow::on_positionReset_14_clicked() { onResetButtonClicked(14); }
+
+void MainWindow::on_positionReset_15_clicked() { onResetButtonClicked(15); }
+
+void MainWindow::on_robotPosition_0_currentIndexChanged(int value) {
```

Comment:
Why do we need a separate function for each of these instead of 1 function that takes in the robotPosition as an argument? Does the UI require these to all be separate functions?

Same for the onpositionreset.

If the UI does require them to all be separate, leave a comment at the top stating this.